modules/simple-payments/simple-payments.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+class Jetpack_Simple_Payments {
+	// These have to be under 20 chars because that is CPT limit.
+	static $post_type_order = 'jp_pay_order';
+	static $post_type_product = 'jp_pay_product';
+
+	static $shortcode = 'simple-payment';
+
+	// Classic singleton pattern:
+	private static $instance;
+	private function __construct() {}
+	static function getInstance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+			self::$instance->register_init_hook();
+		}
+		return self::$instance;
+	}
+
+	private function register_scripts() {
+		wp_register_script( 'paypal-checkout-js', 'https://www.paypalobjects.com/api/checkout.js' );
```

Comment:
I'm not sure if we should delegate the source of this file straight to the Paypal server. I think we should keep a copy of the latest stable version and find a way to update it when it's needed.